ports/winit/minibrowser.rs

```diff
@@ -51,16 +51,21 @@ impl Minibrowser {
                     ui.available_size(),
                     egui::Layout::right_to_left(egui::Align::Center),
                     |ui| {
+                        let response = ui.add_sized(
+                            ui.available_size(),
+                            egui::TextEdit::singleline(&mut *location.borrow_mut()),
+                        );
```

Comment:
can we rename this to something like “location_field”, to make it clear which widget we’re later checking and focusing? also to keep the go button on the right, we need to call this after the ui.button("go") block.

Reply:
Fixed in [3b4c213](https://github.com/servo/servo/pull/30105/commits/3b4c213364d7474377fdeaf3707c87ed79574e04)